mne/viz/evoked.py

```diff
@@ -1476,15 +1500,21 @@ def plot_compare_evokeds(evokeds, picks=list(), gfp=False, colors=None,
     vlines : list of int
         A list of integers corresponding to the positions, in seconds,
         at which to plot dashed vertical lines.
-    ci : float | None
-        If not None and `evokeds` is a [list/dict] of lists, a confidence
-        interval is drawn around the individual time series. This value
-        determines the CI width. E.g., if this value is .95 (the default),
-        the 95% parametric confidence interval is drawn.
-        If None, no shaded confidence band is plotted.
+    ci : float | callable | None
+        If not None and `evokeds` is a [list/dict] of lists, a shaded
```

Comment:
double-backticks on vars (SciPy doesn't follow this convention but I've never seen these actually link properly)